core-bundle/src/Routing/Matcher/UrlMatcher.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Matcher;
+
+use Symfony\Bundle\FrameworkBundle\Routing\RedirectableUrlMatcher;
+use Symfony\Cmf\Component\Routing\NestedMatcher\FinalMatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Routing\RouteCollection;
+
```

Comment:
This needs description, I have no clue what it is supposed to do.

Reply:
description in regards to?

Reply:
What the whole class does.

Reply:
Well it's the UrlMatcher. It extends from the one from the Symfony Framework Bundle, which implements redirecting, and it implements the interface required by Cmf Routing. That's all on the first line of the class, I don't see much else to describe. I don't see a point in adding extensive descriptions to framework functional classes, someone who can't read the code should not care about it anyway.

Reply:
>  I don't see a point in adding extensive descriptions to framework functional classes, someone who can't read the code should not care about it anyway.

This is pure bullshit. But whatever.

Reply:
https://twitter.com/CodeWisdom/status/1090308606409478144